ParamConverter/DataTransferObjectParamConverter.php

```diff
@@ -71,8 +71,9 @@ public function apply(Request $request, ParamConverter $configuration): bool
         $reflectionClass = new \ReflectionClass($configuration->getClass());
 
         $uuid = \uniqid('', false);
+        $parent = $request->attributes->get($configuration->getName()) !== null ? $configuration->getName() : null;
```

Comment:
Please add parenthesis around your condition (and you can maybe make it clearer with variables or at least comments saying what your code does)

Reply:
`!== null` is not necessary, and this could be used:
```suggestion
        $parent = $request->attributes->get($configuration->getName()) ? $configuration->getName() : null;
```